app/views/users/profile.html.erb

```diff
@@ -233,11 +233,16 @@
       </div>
     </div>
   <% end %>
+  <% if current_user.can_moderate? || current_user.role == "admin" %>
```

Comment:
Thanks for the clarification. I am going make changes. I am happy I got to work on this issue

Reply:
@uzorjchibuzor let me know if you need any help! I will be available over the next few hours

Also I have one more change I think you should add:
- You should move the line that checks if there is a current_user above the one that checks if the current_user is a moderator or admin. Otherwise, if there is not a current_user then the 1st method could throw an error